package.json

```diff
@@ -39,6 +43,13 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "FileTreeView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- `*.tflite` or saved model for Tensorflow​\n- `*.onnx` or `*.pth` for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\nNext, register already-installed toolchain or install a new toolchain. You can do this later.\n[Register Installed Toolchain](Command:onevscode.register-installed-toolchain)\nor\n[Install New Toolchain](Command:onevscode.install-new-toolchain)\n",
+        "when": "config.one-vscode.projectRootDirectory == ''"
```

Comment:
For the reason why `config.` is attached to `one-vscode.projectRootDirectory` setting name,
please refer to the following:
- https://code.visualstudio.com/api/references/when-clause-contexts#check-a-setting-in-a-when-clause